### PR TITLE
Enhance selinux_setup: install pkgs explicitly

### PR DESCRIPTION
**Pull request by lilyeyes**
Enhance selinux_setup: install pkgs explicitly
Test cases are passed.

- Related ticket: https://progress.opensuse.org/issues/70090
- Needles: NA
- Verification run: 
  TW: https://openqa.opensuse.org/tests/1382646
  SLES: http://openqa.suse.de/tests/4637370

### CODE REVIEW

File: tests/security/selinux/selinux_setup.pm [hunk 1]
```diff
@@ -33,14 +33,21 @@ sub run {
         assert_script_run('zypper -n in policycoreutils-python');
     }
 
-    my $pkgs = script_output("echo `zypper se selinux | grep -i selinux | grep -v -e srcpackage -e debuginfo -e debugsource -e busybox | cut -d '|' -f 2`");
+    # install as many as SELinux related packages
+    my $pkgs
+      = "selinux-tools libselinux-devel libselinux1 libselinux1-32bit python3-selinux restorecond mcstrans libsepol1 libsepol-devel libsemanage1 libsemanage-devel checkpolicy setools-console";
```
Comment (DimStar77) [inline tests/security/selinux/selinux_setup.pm:38]: I wonder if we should have such an extesnive list - or if we should rather see if enabling SELinux like a user would do it works.

Things like libselinux1 must be part of the dependency chain after all

Comment (lilyeyes) [reply tests/security/selinux/selinux_setup.pm:38]: Thanks for your comments.
The "selinux_setup.pm" is a preparation for "selinux_smoke.pm" and other following test cases.
The extensive list is not so much necessary (at least no harm) but sometime installing some pkgs may make something broken at the beginning of introducing SELinux. So we keep this testing strategy.

**Comment by rfan1** [comment]
Hello Lili,

As we want to install as many selinux related pkgs, can we use wildcard characters rather than list all pkg names here?

**Comment by lilyeyes** [comment]
> Hello Lili,
> 
> As we want to install as many selinux related pkgs, can we use wildcard characters rather than list all pkg names here?

The original method is using wildcard but introduced unnecessary pkgs.

**Comment by rfan1** [comment]
> > Hello Lili,
> > As we want to install as many selinux related pkgs, can we use wildcard characters rather than list all pkg names here?
> 
> The original method is using wildcard but introduced unnecessary pkgs.

Got it, Then it makes sense. (I 'd suggest to modify the abstract for poo as well :))

**Review by rfan1 — APPROVED** [review]
LGTM

**Comment by lilyeyes** [comment]
> > > Hello Lili,
> > > As we want to install as many selinux related pkgs, can we use wildcard characters rather than list all pkg names here?
> > 
> > 
> > The original method is using wildcard but introduced unnecessary pkgs.
> 
> Got it, Then it makes sense. (I 'd suggest to modify the abstract for poo as well :))

Thanks, revised the abstract of the poo.

**Comment by lemon-suse** [comment]
LGTM.

**Pull request merged**